### PR TITLE
Port to Python 3

### LINKED ISSUE #20: Feature request - port to Python 3.6
I've got PlaidML running on my AMD Bonaire on Arch Linux with Python 2.7 in a Conda environment. Every other Python package I have runs with 3.6 and my goal is to keep it that way. ;-)

There doesn't seem to even be a pip package for 3.6, so the `pip install -U plaidml-keras` fails with Python 3.6. If you can post build-from-GitHub-source instructions, I can make a local package and install it.

P.S.: Let me know if you want Arch setup instructions for AMD GPUs. Most of it is on the Arch User Repository wiki but I've got some scripts that do the work.

P.P.S.: Benchmark results
```
Using PlaidML backend.
INFO:plaidml:Initializing device bonaire.0: "Bonaire", vendor "Advanced Micro Devices, Inc."
INFO:plaidml:Opening device "bonaire.0": "Advanced Micro Devices, Inc. Bonaire"
Downloading data from https://github.com/fchollet/deep-learning-models/releases/download/v0.6/mobilenet_1_0_224_tf.h5
16793600/17225924 [============================>.] - ETA: 0s 
Model loaded.
Compiling and running initial batch, batch_size=1
Warmup
Doing the main timing
Example finished, elapsed: 6.821215868 (compile), 15.0223557949 (execution)
```

### PR DESCRIPTION
**Pull request by Juanlu001**
This code should be compatible with Python 2 and 3 at the same time. Fixes #20.

### CODE REVIEW

**Comment by Juanlu001** [comment]
I have never used bazel so I don't really know how to provide Python 2 and 3 buildings. Please feel free to edit accordingly, or care to give pointers.

**Comment by choongng** [comment]
Thanks for the patch! We're still getting our contribution process worked out but will update when ready.

**Comment by znmeb** [comment]
Let me know when this is ready to test - I've gone as far as I can with my limited Python and zero Bazel knowledge.

**Comment by cathalgarvey** [comment]
Folks, Python 2 is dead by 2020, and most new projects and _all_ major ML frameworks now are compatible Python 3. It's very strange to lag on this or consider it a non-priority, I hope this gets merged and maintained properly! :)

**Comment by brianretford** [comment]
@Juanlu001 can you sign the CLA?

**Comment by Juanlu001** [comment]
@brianretford Done, thanks!

**Comment by ikhlestov** [comment]
Guys, I think it will be reasonable to use the [python six module](https://pypi.python.org/pypi/six) - with it you may support both python2 and python3.

**Comment by Juanlu001** [comment]
> Guys, I think it will be reasonable to use the python six module - with it you may support both python2 and python3.

I used python-future instead, see the PR. http://python-future.org/automatic_conversion.html

**Comment by Juanlu001** [comment]
Required reading: https://docs.python.org/3/howto/pyporting.html

**Comment by ikhlestov** [comment]
oh, sorry, I used six too much and missed another approach. Thank you for link - I will learn this also, if such approach will be used as main for the plaidml.

File: plaidml/BUILD [hunk 1]
```diff
@@ -265,5 +265,5 @@ plaidml_py_wheel(
     console_scripts = [
        "'plaidml-setup = plaidml.plaidml_setup:main'",
     ],
-    python = "py2",
+    python = "py3",
```
Comment (rlaverde) [inline plaidml/BUILD]: I think that adding another section for python3 will be better:
```
load(
...   
    "plaidml_py_wheel",
    "plaidml_py3_wheel",
...
)
```

```
plaidml_py3_wheel(
    name = "wheel",
    srcs = [
        ":py",
        ":configs",
    ] + select({
        "//bzl:x64_windows": ["//plaidml:plaidml.dll"],
        "//conditions:default": ["//plaidml:libplaidml.so"],
    }),
    config = ":setup.cfg",
    package = "plaidml",
    platform = select({
        "//bzl:arm7-ubuntu-14.04": "linux_armv7l",
        "//bzl:darwin": "macosx_10_10_x86_64",
        "//bzl:x64_windows": "win_amd64",
        "//conditions:default": "manylinux1_x86_64",
    }),
    console_scripts = [
       "'plaidml-setup = plaidml.plaidml_setup:main'",
    ],
    python = "py3",
)
```

Comment (Juanlu001) [reply plaidml/BUILD]: ¡Gracias @rlaverde! @choongng @brianretford do you approve this approach? I can make these changes myself.

Comment (brianretford) [reply plaidml/BUILD]: This is totally definitely the right approach - we need to be able to build both varieties

Comment (Juanlu001) [reply plaidml/BUILD]: Awesome, go ahead :) Thanks to you for sharing it!

**Comment by brianretford** [comment]
Can you give us edit access to both of your PRs?
https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

**Comment by Juanlu001** [comment]
The "allow edits from maintainers" option is checked from the beginning. Anything else I can do?

**Comment by brianretford** [comment]
Great. I'm good to go. Should have this wrapped up soon.

**Comment by Juanlu001** [comment]
I see the CI is making good progress :grin: I leave it on your hands.

**Pull request merged**